01-node-tutorial/answers/HW_11.js

```diff
@@ -0,0 +1,71 @@
+//11-fs-async.js
+
+const {readFile, writeFile} = require('fs')
+
+
+//without the utf8 we get a buffer reading in the console
+//with the utf8 we printed hello this is the first text file
+//the(err, result) is the callback
+
+readFile('./temporary/subfolder/first.txt', 'utf8',(err, result) => {
```

Comment:
One helpful thing VSCode can do is format the document so you can keep the code more organized.  Right click in the file and click 'format document' and it should fix the spacing and indentation issues.  Otherwise it looks great!